roles/openshift_openstack/tasks/node-configuration.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# NOTE(shadower): we need to do this because some of the install tasks seem to
+# ignore openshift_hostname and rely on the actual system's hostname
```

Comment:
I'd like to clarify this and open a bug for openshift-ansible. AFAIK, that's not the case for AWS provider installations at the very least.